### PR TITLE
mesg: add page

### PR DESCRIPTION
**Pull request by lbonanomi**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

This PR is in support of #2214

### CODE REVIEW

File: pages/linux/mesg.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# mesg
+
+> Check or set a terminal's ability to receive messages from other users, usually from the write command.
+
```
Comment (sbrl) [inline pages/linux/mesg.md]: Maybe we could add an additional like below the description line like this?

> See also `write`.

Comment (lbonanomi) [reply pages/linux/mesg.md]: Would a reciprocal link from write to mesg be appropriate?

Comment (sbrl) [reply pages/linux/mesg.md]: Good idea! Looks like you forget the trailing full stop though

Comment (lbonanomi) [reply pages/linux/mesg.md]: @sbrl I've spent 15 minutes wringing my hands about this: should a change to [write.md](https://github.com/tldr-pages/tldr/blob/master/pages/linux/write.md) be in a separate PR? I realize this is a silly concern, but I'm keen to get it right.

Comment (pxgamer) [reply pages/linux/mesg.md]: @lbonanomi, I would create it as a separate PR. :+1:

Comment (sbrl) [reply pages/linux/mesg.md]: Not a silly question, @lbonanomi! Generally, edits to each command should each have their own PRs. The exception here is if you're making the same change to many pages, such as in #2896 for example.

**Pull request merged**